src/InventiveContent.tsx

```diff
@@ -16,13 +16,14 @@ export const InventiveContent = (props: InventiveContentProps) => {
     if (!urlInfo) return;
 
     const targetOrigin = new URL(urlInfo.url).origin;
+    const reference = window.location.href;
```

Comment:
we don't need to worry about undefined window because it is being called within a useEffect.

Reply:
We are in effect making assumptions on C1's embed/deep-link implementation here.

Instead of grabbing the parent url to use from the browser window, why not let C1 app pass their portion of the url they want our copy-link to use as prop for the component?